interop/src/clients/corecrypto/ffi.rs

```diff
@@ -29,11 +30,11 @@ pub(crate) struct CoreCryptoFfiClient {
     cc: CoreCrypto,
     client_id: Vec<u8>,
     #[cfg(feature = "proteus")]
-    prekey_last_id: u16,
+    prekey_last_id: Cell<u16>,
 }
 
-const KEY_STORE_PATH: &str = "path";
-const KEY_STORE_KEY: &str = "path";
+const KEY_STORE_PATH: &str = "ffi_keystore";
+const KEY_STORE_KEY: &str = "key";
```

Comment:
Nit: this hunk should be moved to the previous commit, since that's the one that introduced the wrong values.